src/app/caesar-portal/profile/profile.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="icon" [ngClass]="{invisible: photoVisibility}" (click)="toggle()">
+    <img [src]="currentUser.image || defaultPhoto" alt="photo">
+</div>
+<aside class="sidebar" [ngClass]="{invisible: popupVisibility}" (mouseleave)="toggle()">
```

Comment:
"Click on the photo should opens the section with the profile data. The section is automatically closed if the mouse focus is lost."